articles/api/authentication/_change-password.md

```diff
@@ -52,9 +54,11 @@ curl --request POST \
 "We've just sent you an email to reset your password."
 ```
 
-Given a user's `email` address and a `connection`, Auth0 will send a change password email.
+Send a change password email to the user with the given email address on the given connection.
 
-This endpoint only works for database connections.
+Optionally, you may provide an Organization ID to support Organization-specific variables in customized email templates and redirect URLs.
+
```

Comment:
^ is there any Org doc we could provide them with the variables or params? This is a good edit! If I was reading it tho, I would want to know where I find those. Just a suggestion tho.

Reply:
Added a link to email template variables. Checked the MGMT API and included more info regarding redirect URLs behavior.